core-api/src/main/resources/definitions/exposed-services.json

```diff
@@ -604,6 +604,20 @@
       "visibleForDatalake": true,
       "visibleForDatahub": true
     },
+    {
+      "name": "QUERY_PROCESSOR",
+      "displayName": "Hue Query Processor",
+      "serviceName": "QUERY_PROCESSOR",
+      "knoxService": "QUERY_PROCESSOR",
+      "knoxUrl": "/query_processor/",
+      "ssoSupported": true,
+      "port": 30700,
+      "tlsPort": 30700,
+      "apiOnly": false,
+      "apiIncluded": false,
+      "visibleForDatalake": true,
+      "visibleForDatahub": true
```

Comment:
could you please validate are these the right values? It can be accessed via URL? Accessible for datalake and datahub as well?